docs/web/user_guide.md

```diff
@@ -332,17 +332,17 @@ root account arguments:
                         not require authentication otherwise.
 ```
 
-#### Enfore secure socket (SSL)
+#### Enfore secure socket (TLS)
```

Comment:
I'd not **replace** SSL with TLS but if we want to mention TLS then let's mention both. The terminology common to many people involved in running servers and web connection is still "SSL", and not saying "TLS", no matter how wrong this is. If people are just searching the documentation how to use SSL, they will no longer have findings related to this (to some very important) feature.